### ISSUE TITLE
False errors appear in source file after editing it

### ISSUE THREAD

**Issue by skalinets** [ionide/ionide-vscode-fsharp#342]
After editing of fs file it gets highlighted with errors, but still compiles.

## Steps

1. Open vscode. 
1. Create new expecto project using Ionide, build it
1. Open `Tests.fs` 
    - Note that no errors are highlighted
1. Change `open Expecto` to `open Expect`
    - Errors appear (as expected)
1. Revert your change. 
    - Errors are still there. Autocompletion is broken.

## Screen

![image](https://cloud.githubusercontent.com/assets/822748/22149420/6163f9f8-df1c-11e6-91e1-086975c11e9f.png)

## Logs
[ionide-error.log.txt](https://github.com/ionide/ionide-vscode-fsharp/files/719342/ionide-error.log.txt)

## Versions
VS CODE         1.8.1   
Ionide-fsharp  2.19.5

**Comment by Krzysztof-Cieslak** [comment]
```
[14:19:35 DEBUG] RES (002) <- {project} in 898 ms: Kind={"error"}
Data="Expecting element 'root' from namespace ''.. Encountered 'None'  with name '', namespace ''. "
```

Looks like there is some problem with fsproj file, can you post it too.

**Comment by skalinets** [comment]
here it is https://gist.github.com/skalinets/9dfe09092161c89c0daf517fbbf39647

**Comment by skalinets** [comment]
But the project was generated by Ionide / forge. I did not touch anything there.

**Comment by Krzysztof-Cieslak** [comment]
Yeah, I can't see anything suspicious there.

In `%USERPROFILE%\.vscode\extensions\Ionide.Ionide-fsharp-2.19.5\bin` there should be `FSharp.Compiler.Service.ProjectCrackerTool.exe`

Try to run `FSharp.Compiler.Service.ProjectCrackerTool.exe --text <path_to>/<my_project>.fsproj true`

**Comment by skalinets** [comment]
```
C:\Users\admin\.vscode\extensions\Ionide.Ionide-fsharp-2.19.5\bin> .\FSharp.Compiler.Service.ProjectCrackerTool.exe --text C:\Users\admin\work\fst\exptests\exptests.fsproj
{ProjectFile = "";
 Options = [||];
 ReferencedProjectOptions = [||];
 LogOutput = "At least two arguments required.";}
C:\Users\admin\.vscode\extensions\Ionide.Ionide-fsharp-2.19.5\bin> .\FSharp.Compiler.Service.ProjectCrackerTool.exe --text C:\Users\admin\work\fst\exptests\exptests.fsproj true

Process is terminated due to StackOverflowException.
```

**Comment by Krzysztof-Cieslak** [comment]
You have VS2017 RC installed by any chance? I guess it's duplicate of https://github.com/ionide/ionide-vscode-fsharp/issues/291

**Comment by skalinets** [comment]
yes, I do have it installed. 
Regarding duplicate -- #291 refers to https://github.com/fsharp/FSharp.Compiler.Service/issues/672 that in turn gets reproduced when project filename contains dots. Mine does not.

I just checked my VS2017 and it seems have new update *15.0.26014.0 --> 15.0.26020.0*. Will give it a chance :)

**Comment by skalinets** [comment]
OK, I uninstalled VS2017 and issue has gone. I think this issue can be closed as a duplicate of #291. Thanks.
